.github/workflows/csharp.yml

```diff
@@ -31,7 +31,7 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
-    run-tests:
+    build-and-test-c#-client:
```

Comment:
please split out the rename calls.  that way if AMZ wants to revert the -latest changes, they can revert the whole PR